src/Raven.Server/Documents/DocumentTombstoneCleaner.cs

```diff
@@ -168,6 +168,6 @@ public override int Execute(DocumentsOperationContext context)
 
     public interface IDocumentTombstoneAware
     {
-        Dictionary<string, long> GetLastProcessedDocumentTombstonesPerCollection();
+        IReadOnlyDictionary<string, long> GetLastProcessedDocumentTombstonesPerCollection();
```

Comment:
Why did you make this change? This is now forcing us to make virtual interface calls, instead of direct calls.
Note that inside `Raven.Server` we don't need to make any encapsulation of types.